### DISCUSSION TITLE
Auth0 Login Rules

### DISCUSSION THREAD

**Discussion by Maxim270693** [lowdefy/lowdefy#1588] [Questions and Help] [answered]
please tell me how to call asynchronous function from js file

I want to call on click Sign in

example taken from https://github.com/lowdefy/lowdefy-example-openid-connect/blob/main/auth0/loginRule.js but the function is not used anywhere

![image](https://github.com/lowdefy/lowdefy/assets/77190011/081017d4-afcb-4541-b0e4-da0a05673e39)
![image](https://github.com/lowdefy/lowdefy/assets/77190011/0effd9dc-1589-4906-a312-ea9ed7b27b72)

Thank you

**Comment by SamTolmay** [comment, marked as answer]
Hi @Maxim270693 

This function is not used by Lowdefy, it is configured in the Auth0 platform as a login rule, which you can read more about here: https://auth0.com/docs/customize/rules

**Comment by Maxim270693** [comment]
@SamTolmay Thank you
